front/src/components/Common/Footer/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import './footer.scss'
+
+const Footer = () =>{
```

Comment:
Please write down a task for a future: add eslint to the project. Why? To help yourself and fix code style issues. Space should be placed between => and {